XMLtoMD/GuidelineXmlToMD/MarkdownOut/MdText.cs

```diff
@@ -0,0 +1,220 @@
+        /// </summary>
+        public static readonly string BoldWrap = "**";
+
+        /// <summary>
+        /// The Markdown code string to be wrapped around a string of text.
+        /// </summary>
+        public static readonly string CodeWrap = "`";
+
+        /// <summary>
+        /// The Markdown strike-through string to be wrapped around a string of text.
+        /// </summary>
+        public static readonly string StrikeThroughWrap = "~~";
+
+        /// <summary>
+        /// The Markdown heading 1 prefix string.
+        /// </summary>
+        public static readonly string Heading1Prefix = "# ";
+
+        /// <summary>
+        /// The Markdown heading 2 prefix string.
+        /// </summary>
+        public static readonly string Heading2Prefix = "## ";
+
+        /// <summary>
+        /// The Markdown heading 3 prefix string.
+        /// </summary>
+        public static readonly string Heading3Prefix = "### ";
+
+        /// <summary>
+        /// The Markdown heading 4 prefix string.
+        /// </summary>
+        public static readonly string Heading4Prefix = "#### ";
+
+        /// <summary>
+        /// The Markdown heading 5 prefix string.
+        /// </summary>
+        public static readonly string Heading5Prefix = "##### ";
+
+        /// <summary>
+        /// The Markdown heading 6 prefix string.
+        /// </summary>
+        public static readonly string Heading6Prefix = "###### ";
+
+        /// <summary>
+        /// The Markdown quote prefix string.
+        /// </summary>
+        public static readonly string QuotePrefix = "> ";
+
+        /// <summary>
+        /// The Markdown unordered list item prefix string.
+        /// </summary>
+        public static readonly string UnorderedListItemPrefix = "- ";
+
+
+        /// <summary>
+        /// The default number used for ordered list items.
+        /// </summary>
+        public static readonly int DefaultListItemNumber = 1;
+
+        /// <summary>
+        /// The Markdown ordered list item prefix string.
+        /// </summary>
+        public static readonly string OrderedListItemPrefix = DefaultListItemNumber + ". ";
+
+        /// <summary>
+        /// The Markdown indent string for sublists.
+        /// </summary>
+        public static readonly string ListItemIndent = new string(' ', 5);
+
+        /// <summary>
+        /// Styles the provided text using the specified <see cref="MdStyle"/>.
+        /// </summary>
+        /// <param name="text">The text to style.</param>
+        /// <param name="style">The Markdown style to apply.</param>
+        /// <returns>The styled string.</returns>
+        /// <exception cref="System.ArgumentException">The style is not recognized.</exception>
+        public static string Style(object text, MdStyle style) {
+            string wrap;
+            switch (style) {
+                case MdStyle.None: wrap = String.Empty; break;
```

Comment:
nit: Use keyword over BCL type
```suggestion
                case MdStyle.None: wrap = string.Empty; break;
```